src/AllCountries.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from "react";
+import Card from "./Card";
+import SidebarWidth from "./Sidebar";
+import { FilterOptions } from "./ConstantData";
+
+export default function AllCountries() {
+  const ApiCall = async () => {
```

Comment:
Please change the function name to reflect what it is doing. For example, since it is fetching the countries data, you can name it fetchCountriesData for something like that.